ui/components/app/snaps/snap-authorship-header/snap-authorship-header.js

```diff
@@ -35,14 +35,15 @@ const SnapAuthorshipHeader = ({
   const snapPrefix = snapId && getSnapPrefix(snapId);
   const packageName = snapId && removeSnapIdPrefix(snapId);
   const isNPM = snapPrefix === 'npm:';
-  const url = isNPM
-    ? `https://www.npmjs.com/package/${packageName}`
-    : packageName;
 
   const subjectMetadata = useSelector((state) =>
     getTargetSubjectMetadata(state, snapId),
   );
 
+  const url = isNPM
+    ? `https://www.npmjs.com/package/${packageName}/v/${subjectMetadata?.version}`
```

Comment:
should we add a check if the version is null before we append it to the url?

Reply:
done